lib/reporters/tap.js

```diff
@@ -32,6 +32,7 @@ function TAP(runner) {
   var failures = 0;
 
   runner.on('start', function() {
+    console.log('TAP version 13');
```

Comment:
Consider using `self.write()` and `self.writeln()` so that file-baseed output could be a possibility.

Reply:
xunit reporter has a `write()` and I find nothing about `writeln()`. Can you point me in the right direction? `console.log` is what most reporters seem to be doing. :scream:

Reply:
Sorry, using functions from another proposed PR.
Until that's ready, you could implement those as:
```js
/**
 * Write out the given string.
 *
 * @param {string} s
 */
TAP.prototype.write = function(s) {
  if (this.fileStream) {
    this.fileStream.write(s);
  } else {
    process.stdout.write(s);
  }
};

/**
 * Write out the given string and terminate the line.
 *
 * @param {string} s
 */
TAP.prototype.writeln = function(s) {
  this.write(s + '\n');
};
```

Reply:
Glancing your `console.log` usage, will also need to add an `sprintf()` function to handle your string substitutions.